src/test/java/com/github/javafaker/service/FakeValuesServiceTest.java

```diff
@@ -234,7 +234,7 @@ public void expressionWithValidFakerObjectValidMethodInvalidArgs() {
 
     @Test
     public void futureDateExpression() throws ParseException {
-        SimpleDateFormat dateFormat = new SimpleDateFormat( "EEE MMM dd HH:mm:ss z yyyy" );
+        SimpleDateFormat dateFormat = new SimpleDateFormat( "EEE MMM dd HH:mm:ss z yyyy", new Locale("test") );
```

Comment:
why is this change part of the PR?

Reply:
Because test cannot pass, if your default locale is not EN. You can exclude this commit if you want, but I am not able to run tests without this line on my machine.

Reply:
Yes I'd prefer not to have this change as part of this PR.

Reply:
OK, I removed it.